modules/installation-azure-limits.adoc

```diff
@@ -184,6 +184,14 @@ ifndef::ash[]
 endif::ash[]
 |The internal load balancer, each of the three control plane machines, and each
 of the three worker machines each use a private IP address.
+
+
+|Spot VMs
+|
```

Comment:
Is it possible to provide information here rather than leave it blank?

Reply:
@maxwelldb There is no default value for spot VMs.